Source/WebKit/UIProcess/ViewGestureController.cpp

```diff
@@ -408,6 +408,16 @@ static bool deltaShouldCancelSwipe(FloatSize delta)
     return std::abs(delta.height()) >= std::abs(delta.width()) * minimumScrollEventRatioForSwipe;
 }
 
+const char* ViewGestureController::PendingSwipeTracker::stateToString(State state) const
```

Comment:
can/should this be static?

Reply:
also, can't this be overloaded `operator <<` instead?

Reply:
Maybe if that can access the private enum in a nested class?